lib/test/cdo/test_mailjet.rb

```diff
@@ -154,6 +156,31 @@ def test_create_contact_and_send_welcome_email
     MailJet.create_contact_and_send_welcome_email(user)
   end
 
+  def test_send_cap_section_warning_email
+    email = 'fake.email@test.xx'
+
+    sign_up_time = Time.now.to_datetime
+
+    user = mock
+    user.stubs(:id).returns(1)
+    user.stubs(:email).returns(email)
+    user.stubs(:name).returns('Fake Name')
+    user.stubs(:teacher?).returns(true)
+    user.stubs(:created_at).returns(sign_up_time)
+
+    sections = {'sections' => [
```

Comment:
Should this be `'sections'` or `'capSections'`? The documentation is not consistent with the test.